CONTRIBUTING.md

```diff
@@ -151,3 +151,22 @@ Here are some links to help guide you on style as recommended by Dart:
 * Design: <https://dart.dev/guides/language/effective-dart/design>
 
 We recommend following these same guidelines for any of your own packages you may create for the ecosystem, as well.
+
+## Adding a New Component
```

Comment:
We should update this section for the new generation flow with the configurator

Reply:
Actually, I copied this section into another branch and updated it based on recent developments, so we can remove it from this PR.